index.html

```diff
@@ -14,172 +14,200 @@
     <div class="wrapper">
         <header class="nav-bar">
             <a href="#">
-                <p>Flex Box</p>
+                <p>Hamma Azarok</p>
             </a>
             <nav> <img src="img/Enabled.png" alt="Logo"></nav>
         </header>
-        <section class="header">
-            <h1 class="header-text">I’m Hamma
-                Glad to see you!</h1>
-            <p>
-                I’m a software developer! I can help you build a product , feature or website Look through some of my
-                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
-                contact me.
-            </p>
-            <a href="#">LET’S CONNECT</a>
-            <ul class="social-links">
-                <li><a href="#"><img src="img/icons/Vector.png" alt=""></a></li>
-                <li><a href="#"><img src="img/icons/Linkedin_icon.png" alt=""></a></li>
-                <li><a href="#"><img src="img/icons/Vector-1.png" alt=""></a></li>
-                <li> <a href="#"><img src="img/icons/Vector-2.png" alt=""></a></li>
-                <li> <a href="#"><img src="img/icons/Vector3.png" alt=""></a></li>
+        <nav class="nav-bar-desktop">
+            <a class="logo" href="#">Hamma Azarok</a>
+            <ul class="nav-list">
+                <li class="nav-item"><a href="#portfolio">Portofolio</a></li>
+                <li class="nav-item"><a href="#about">About</a></li>
+                <li class="nav-item"><a href="#contact">Contact</a></li>
             </ul>
+        </nav>
```

Comment:
- [ ] there is a space at top of your header it would be better if you remove it in order to match the design 
![image](https://user-images.githubusercontent.com/61790587/171682854-2a75f1e4-1c83-4727-9f63-858b58809cbd.png)